lib/presentation/pages/main/login/login_page.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter_awesome/presentation/misc/app_styles.dart';
+import 'package:flutter_awesome/presentation/pages/main/login/login_controller.dart';
+import 'package:flutter_awesome/presentation/widgets/asset_widget.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class LoginPage extends BasePage<LoginController> {
+  LoginPage({Key? key, required super.controller}) : super(key: key);
+
+  static const String route = 'Login';
+
+  @override
+  Widget get view =>
+      ControlledWidgetBuilder<LoginController>(builder: (context, controller) {
+        return Scaffold(
+          key: globalKey,
+          body: SingleChildScrollView(
+            child: Container(
+              padding: EdgeInsets.only(top: AppSize.dimen24.w),
+              child: Container(
+                padding: const EdgeInsets.symmetric(
+                  horizontal: AppSize.dimen20,
+                  vertical: AppSize.dimen32,
+                ),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    TSImage.asset(assetName: 'img_login.png'),
+                    SizedBox(height: AppSize.dimen46.w),
+                    Text(
+                      'Silahkan masuk dengan nomor telkomsel kamu',
+                      style: textStyleW700S18,
+                    ),
+                    SizedBox(height: AppSize.dimen24.w),
+                    Text('Nomor HP', style: textStyleW700S14),
+                    TextField(
+                      onChanged: (value) {
+                        controller.handleChangeNumber(value);
+                      },
+                      style: textStyleW500S14,
+                      decoration: InputDecoration(
+                          hintText: "Cth. 08129011xxxx",
+                          hintStyle:
+                              textStyleW500S14.copyWith(color: colorTextHint),
+                          contentPadding: const EdgeInsets.all(12.0),
+                          border: const OutlineInputBorder(
+                            borderSide: BorderSide(color: colorBlack),
+                          ),
+                          focusedBorder: const OutlineInputBorder(
+                            borderSide: BorderSide(color: colorBlack),
+                          )),
+                    ),
+                    SizedBox(height: AppSize.dimen16.w),
+                    Row(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Checkbox(
+                          materialTapTargetSize:
+                              MaterialTapTargetSize.shrinkWrap,
+                          activeColor: colorPrimary,
+                          value: controller.checkbox,
+                          onChanged: (_) {
+                            controller.handleCheckbox();
+                          },
```

Comment:
Prefer using fat arrow to call function, because the method doesn't using parameter. please change this.
e.g : 
```
onChanged: () => controller.method,
```